v2/internal/system/packagemanager/apt.go

```diff
@@ -59,8 +62,13 @@ func (a *Apt) PackageInstalled(pkg *Package) (bool, error) {
 	if pkg.SystemPackage == false {
 		return false, nil
 	}
-	stdout, _, err := shell.RunCommand(".", "apt", "-qq", "list", pkg.Name)
-	return strings.Contains(stdout, "[installed]"), err
+	cmd := exec.Command("apt", "-qq", "list", pkg.Name)
+	var stdo, stde bytes.Buffer
+	cmd.Stdout = &stdo
+	cmd.Stderr = &stde
+	cmd.Env = append(os.Environ(), "LANG=en_US.utf8")
```

Comment:
please change line 69 to
```go
cmd.Env = append(os.Environ(), "LC_ALL=en_US.utf8")
```
```log
$ locale
LANG=en_US.UTF-8
LANGUAGE=
LC_CTYPE="fr_FR.UTF-8"
LC_NUMERIC="fr_FR.UTF-8"
LC_TIME="fr_FR.UTF-8"
LC_COLLATE="fr_FR.UTF-8"
LC_MONETARY="fr_FR.UTF-8"
LC_MESSAGES="fr_FR.UTF-8"
LC_PAPER="fr_FR.UTF-8"
LC_NAME="fr_FR.UTF-8"
LC_ADDRESS="fr_FR.UTF-8"
LC_TELEPHONE="fr_FR.UTF-8"
LC_MEASUREMENT="fr_FR.UTF-8"
LC_IDENTIFICATION="fr_FR.UTF-8"
LC_ALL=fr_FR.UTF-8
```
```
$ wails doctor
Wails CLI v2.0.0-beta.yo


Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:                     Ubuntu
Version:                20.04
ID:                     ubuntu
Go Version:             go1.17.1
Platform:               linux
Architecture:           amd64
Package Manager:        apt

Dependency      Package Name            Status          Version
----------      ------------            ------          -------
*docker         docker.io               Installed       20.10.12
gcc             build-essential         Installed       12.8ubuntu1.1
libgtk-3        libgtk-3-dev            Installed       3.24.20-0ubuntu1
libwebkit       libwebkit2gtk-4.0-dev   Installed       2.34.4-0ubuntu0.20.04.1
npm             npm                     Installed       8.3.1
pkg-config      pkg-config              Installed       0.29.1-0ubuntu4

* - Optional Dependency

Diagnosis
---------
Your system is ready for Wails development!
```
https://stackoverflow.com/questions/43953328/how-to-set-locale-in-the-current-terminals-session